airflow/models/baseoperator.py

```diff
@@ -462,10 +461,12 @@ def __new__(cls, name, bases, namespace, **kwargs):
 @functools.total_ordering
 class BaseOperator(AbstractOperator, metaclass=BaseOperatorMeta):
     """
-    Abstract base class for all operators. Since operators create objects that
-    become nodes in the dag, BaseOperator contains many recursive methods for
-    dag crawling behavior. To derive this class, you are expected to override
-    the constructor as well as the 'execute' method.
+    Abstract base class for all operators.
+
+    Since operators create objects that become nodes in the dag, BaseOperator
+    contains many recursive methods for dag crawling behavior. To derive this
+    class, you are expected to override the constructor as well as the 'execute'
+    method.
```

Comment:
```suggestion
    Since operators create objects that become nodes in the DAG, BaseOperator
    contains many recursive methods for DAG crawling behavior. To derive from
    this class, you are expected to override the constructor and the 'execute'
    method.
```